ext/pgsql/pgsql.c

```diff
@@ -281,11 +281,11 @@ static void _free_result(zend_resource *rsrc)
 }
 /* }}} */
 
-static int _php_pgsql_detect_identifier_escape(const char *identifier, size_t len) /* {{{ */
+static bool _php_pgsql_detect_identifier_escape(const char *identifier, size_t len) /* {{{ */
```

Comment:
This function could use a rename... maybe `_php_pgsql_identifier_is_escaped()` or `_php_pgsql_identifier_requires_escape()` if you invert the semantics.